hypothesis-python/src/hypothesis/database.py

```diff
@@ -339,13 +339,21 @@ def move(self, src: bytes, dest: bytes, value: bytes) -> None:
 
 class GitHubArtifactDatabase(ExampleDatabase):
     """
-    A file-based database loaded from a GitHub Actions artifact.
+    A file-based database loaded from a `GitHub Actions <https://docs.github.com/en/actions>`_ artifact.
 
     You can use this for sharing example databases between CI runs and developers, allowing
     the latter to get read-only access to the former. This is particularly useful for
-    continuous fuzzing (i.e. with `HypoFuzz <https://hypofuzz.com/>`__),
-    where the CI system can upload the resulting database as an artifact after each run,
-    allowing developers to reproduce any failures by just running hypothesis.
+    continuous fuzzing (i.e. with `HypoFuzz <https://hypofuzz.com/>`_),
+    where the CI system can help find new failing examples through fuzzing,
+    and developers can reproduce them locally without any manual effort.
+
+    .. note::
+        You must provide `GITHUB_TOKEN` as an environment variable. In CI, Github Actions provides
```

Comment:
```suggestion
        You must provide ``GITHUB_TOKEN`` as an environment variable. In CI, Github Actions provides
```

`.rst` needs double-backticks for monospace text - this _still_ gets me occasionally 😅